feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/component/TimetableGridItem.kt

```diff
@@ -124,6 +152,50 @@ fun TimetableGridItem(
     }
 }
 
+/**
+ *
+ * Calculate the font size of the title by the height of the displayed session grid.
+ *
+ * @param textStyle session title text style.
+ * @param localDensity local density.
+ * @param gridItemHeightPx session grid item height. (unit is px.)
+ * @param speaker session speaker.
+ * @param titleLength session title length.
+ *
+ */
+private fun calculateTitleFontSize(
+    textStyle: TextStyle,
+    localDensity: Density,
+    gridItemHeightPx: Int,
+    speaker: TimetableSpeaker?,
+    titleLength: Int,
+): TextUnit {
+    val titleToScheduleSpaceHeight = with(localDensity) { 4.dp.toPx() }
```

Comment:
Can we make it const like `private val titleToScheduleSpaceHeight =..`? to maintain this 👀 
Or I'm thinking if we can use `Layout()` composable. Did you think it?

Reply:
> Can we make it const like private val titleToScheduleSpaceHeight =..? to maintain this 👀
> I'm thinking if we can use Layout() composable. Did you think it? This could be rather complex so I'm not sure it is better 👀

I see !

Layout composable can certainly be implemented, but I haven't touched it, so it looks like it will take a lot of time.

So, for the time being, we will make the size used for layout a constant while keeping the current implementation.
And, I will try a little with reference to the following documents and blogs.

- https://developer.android.com/jetpack/compose/layouts/custom#create-custom
- https://at-sushi.work/blog/52/

Reply:
We've made it possible to manage with constants!

I am referring to TimetableSizes.

https://github.com/DroidKaigi/conference-app-2023/pull/563/commits/a1f29bebdf64afbdbd44e5b3c27d0c0126ada31d